### PR TITLE
Signuppage/onboarding

### PR DESCRIPTION
**Pull request by giorgio40**
Pull request for login and register spinner added

### CODE REVIEW

**Review by Koyamot — CHANGES_REQUESTED** [review]
Congrats on getting the loader working! Before we merge, let's make sure the form's input fields are correct. Will you please check each field name/placeholder?

[Preview URL](https://safeforum-git-signuppage-onboarding-safeforum.vercel.app/register)

File: src/pages/login.js [hunk 1]
```diff
@@ -33,17 +40,19 @@ function Login(props) {
     update(proxy, { data: { login: userData } }) {
         console.log("This is user payload", userData)
       context.login(userData);
-      navigate("/");
+      navigate("/dashoard");
```
Comment (Koyamot) [inline src/pages/login.js]: There is a typo here.

File: src/pages/register.js [hunk 2]
```diff
@@ -73,170 +81,160 @@ const Register = (props) => {
...
-              id="firstName"
-              className="w-full p-2.5 ml-2 bg-transparent outline-none"
-              onChange={onChange}
-              required
-            />
-          </div>
+          <input
+            name="firstName"
+            type="text"
+            placeholder="firstName"
```
Comment (Koyamot) [inline src/pages/register.js]: Check all inputs for correct spelling. `placeholder="first name"`

File: src/pages/register.js [hunk 3]
```diff
@@ -73,170 +81,160 @@ const Register = (props) => {
...
+            className="w-full mt-2 px-3 py-2 text-gray-500 bg-transparent outline-none border focus:border-indigo-600 shadow-sm rounded-lg"
+            onChange={onChange}
+            required
+          />
         </div>
+
         <div className="flex justify-center">
-          <button className="px-5 py-3 text-white duration-150 bg-indigo-600 rounded-lg hover:bg-indigo-700 active:shadow-lg">
-            Signup
+          <button className="mt-5 px-6 py-2 text-white duration-150 bg-indigo-600 rounded-lg hover:bg-indigo-700 active:shadow-lg">
```
Comment (Koyamot) [inline src/pages/register.js:236]: Let's scrap the all caps and use "Remove" instead

File: src/pages/register.js [hunk 4]
```diff
@@ -73,170 +81,160 @@ const Register = (props) => {
...
+       {/* {isLoading ? <Loading /> : <Dashboard />}  */}
+
+      <img
+        src="https://floatui.com/logo.svg"
+        width={150}
+        className="mx-auto"
+        alt="logo"
+      />
+      <div className="text-center font-semibold font-serif mt-4">REGISTER</div>
       <form onSubmit={onSubmit}>
```
Comment (Koyamot) [inline src/pages/register.js]: Let's add a 40px margin to the bottom of form.

**Review by Koyamot — CHANGES_REQUESTED** [review]

File: src/pages/register.js (hunk 3 above)
Comment (giorgio40) [reply src/pages/register.js:236]: Remove what'?

File: src/pages/register.js [hunk 5]
```diff
@@ -73,170 +81,160 @@ const Register = (props) => {
...
     <>
+       {/* {isLoading ? <Loading /> : <Dashboard />}  */}
+
+      <img
+        src="https://floatui.com/logo.svg"
+        width={150}
+        className="mx-auto"
+        alt="logo"
+      />
+      <div className="text-center font-semibold font-serif mt-4">REGISTER</div>
```
Comment (Koyamot) [inline src/pages/register.js]: Change from REGISTER to Register

**Review by Koyamot — APPROVED** [review]
Good work!

**Pull request merged**